src/components/NoticeBar/NoticeBar.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div
+    class="font-heading font-semibold text-base text-center leading-tight bg-yellow text-dark-blue py-6"
+  >
+    <!-- @slot Content goes here -->
+    <slot />
+  </div>
+</template>
+
+<script>
+/**
+ * Displays notices is a yellow strip. It on
```

Comment:
Whoops, looks like this comment is incomplete 🙂

Reply:
Thanks for catching that 🤦. cc4d8f4 adds the rest of the documentation.